lib/matplotlib/backend_tools.py

```diff
@@ -99,20 +98,6 @@ def trigger(self, sender, event, data=None):
 
         pass
 
-    @figure.setter
-    def figure(self, figure):
-        """
-        Set the figure
-
-        Set the figure to be affected by this tool
-
-        Parameters
-        ----------
-        figure: `Figure`
-        """
-
-        self._figure = figure
```

Comment:
Not sure how much MEP22 modifications I want to go into the MEP27 branch, I think we should keep it to a bare minimum of changes where:
a) overlap exists (i.e. the method calls between the two), and
b) API changes for interaction with the backends.

I.e. Just the things we consider we need to get right before we can start converting the other backends, because I see this main PR as the slowest link in the chain.